src/tv2-constants/tv2-playout-content.ts

```diff
@@ -0,0 +1,159 @@
+import { DVESources, findSourceInfo, ShowStyleContext, SourceDefinition, SourceInfo, SourceInfoType } from 'tv2-common'
+
+export enum PlayoutContentType {
```

Comment:
Thanks for having the same order across repos, something in me just really appreciated that :-D

Reply:
You can thank good old "ctrl+c, ctrl+v" ;)